jenkins/opensearch/perf-test.jenkinsfile

```diff
@@ -0,0 +1,153 @@
+lib = library(identifier: "jenkins@20211118", retriever: legacySCM(scm))
+
+pipeline {
+    agent none
+     options {
+        timeout(time: 10, unit: 'HOURS')
+    }
+    environment {
+        AGENT_X64 = 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+        AGENT_IMAGE = 'opensearchstaging/ci-runner:ci-runner-centos7-v1'
+        BUNDLE_MANIFEST = "bundle-manifest.yml"
+    }
+    parameters {
+        string(
+            name: 'GITHUB_TOKEN',
+            description: 'Github token for account access.',
+            trim: true
+        )
+        string(
+            name: 'BUNDLE_MANIFEST_URL',
+            description: 'The bundle manifest URL, e.g. https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/1.2.2/98/linux/x64/builds/opensearch/manifest.yml.',
+            trim: true
+        )
+        string(
+            defaultValue: 'nyc_taxis',
+            name: 'TEST_WORKLOAD',
+            description: 'The agent label where the tests should be executed, e.g. Jenkins-Agent-al2-x64-c54xlarge-Docker-Host.',
+            trim: true
+        )
+        string(
+            defaultValue: '1',
+            name: 'TEST_ITERATIONS',
```

Comment:
What will happen if I enter a string instead of a number for this? Will `test.sh` fail and fail the job? If no, we need checks so the node is not stuck, else we are good.

Reply:
@kotwanikunal - Can you please confirm? Thank you

Reply:
I have already updated the Jenkins check to validate its a number. But I also did verify the test fails if we execute the script with a non number otherwise.

Reply:
Thanks!